src/VCEL.Core/Helper/Constants.cs

```diff
@@ -3,5 +3,6 @@
     public static class Constants
     {
         public const string DefaultContext = "vcelContext";
+        public const string ViewClientContext = "viewclientContext";
```

Comment:
`underscoreContext` or just `_`

VCEL does not know about ViewClient

Reply:
renamed to `underscoreContext` because `_._.env.user` doesn't look as nice

Reply:
Updated to not include any of them. So `_.env.user` will still be `_.env.user`